client/src/Views/ReportsPage/AllShouts.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-tab-item class="all-container">
```

Comment:
I don't think that this needed to have changed...seems to still work with `v-tab-item`